_posts/2022-06-21-share-bold-ideas

```diff
@@ -0,0 +1,20 @@
+
+Fast forward to 2 years later and I came up with this bold crazy idea to develop a video game for my high school. I called it B-Town Vice City. The concept was simple create a game very much like the much famed (Vice City)[https://en.wikipedia.org/wiki/Grand_Theft_Auto:_Vice_City] at the time with the setting as my  high school. I had less than a 500 hours programming experience. I knew nothing about video games or 3d modeling. Or anything for that matter. I told a bunch of guys in the computer club who also thought it was a bold idea but they liked it so we got to work trying to learn everything there was about making video games. 
+
+Word slowly went out that Bubu and the Computer Club were making a video game; B-Town Vice City. I became a laughing stock. Everywhere I went, people pointed fingers at me and laughed at me; That is the guy who wants to build Vice City. It became so bad that some guys in the club started to distance themselves from the effort because we had become the latest laughing stock. People would see me and ask me how my game was going basically to elicit laughter from their group of friends basically at my expense. It bothered me at first but nonetheless I persisted. I slowly began to grow a thick skin. I dont know where the conviction came from but I was set on developing a video game and nothing anyone would say would deter me. And for a long time nothing did. Now that I look back I think I would probably have laughed at me back then. 
+
+Those of us who were left on the effort decided to abandon the mission due to reasons that had nothing to do with the fact that we were laughed at or that the idea was too audacious. We went on to do different things. But for years the laughter followed. Even past university. Even today, some mates from high school see me today and casually remind me about how I dared to build a game 14 years ago. But these days it is more in nostalgia than it is in ridicule. So we laugh. 
+
+I have since gone on to share bold ideas where ever i find mself. A lot of people laugh but soon stop when they realize oh shit, he is serious. I won't lie, majarity of my ideas are ridicoulous, if not bat shit crazy. But that doesn't stop me. Because a while ago, I realized the power or sharing your ideas. Your crazy idea might not get considered, but nobody will ever forget that you were the one who dared share them in the first place. And this alone counts for everything. And if any of your crazy ideas ever do get taken seriously, it is almost always a thrill. 
```

Comment:
```suggestion
I have since gone on to share bold ideas where ever I find myself. A lot of people laugh but soon stop when they realize oh shit, he is serious. I won't lie, the majority of my ideas are ridiculous, if not batshit crazy. But that doesn't stop me. Because a while ago, I realized the power of sharing your ideas. Your crazy idea might not get considered, but nobody will ever forget that you were the one who dared share them in the first place. And this alone counts for everything. And if any of your crazy ideas ever do get taken seriously, it is almost always a thrill. 
```